netbox/users/migrations/0005_alter_user_table.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 5.0.1 on 2024-01-31 23:18
+
+from django.db import migrations
+
+
+def update_content_types(apps, schema_editor):
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+    # Delete the new ContentTypes effected by the new models in the users app
+    ContentType.objects.filter(app_label='users', model='user').delete()
+
+    # Update the app labels of the original ContentTypes for auth.User to ensure
+    # that any foreign key references are preserved
+    ContentType.objects.filter(app_label='auth', model='user').update(app_label='users')
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0002_squashed_0004'),
+    ]
+
+    operations = [
+        migrations.AlterModelTable(
+            name='user',
+            table=None,
```

Comment:
I assume setting `table=None` reverts to Django's default name resolution for the table (i.e. `users_user`)? If so, we should add a quick note explaining this as it's a bit confusing without context. (And if not I'm lost :laughing:)

Reply:
Not sure? Was from a auto created migration when changed db_table value on the model...